_includes/v23.1/misc/session-vars.md

```diff
@@ -22,6 +22,7 @@
 | <a name="enable-insert-fast-path"></a> `enable_insert_fast_path` | Indicates whether CockroachDB will use a specialized execution operator for inserting into a table. We recommend leaving this setting `on`. | `on` | Yes | Yes |
 | <a name="enable-zigzag-join"></a> `enable_zigzag_join` | Indicates whether the [cost-based optimizer](cost-based-optimizer.html) will plan certain queries using a zig-zag merge join algorithm, which searches for the desired intersection by jumping back and forth between the indexes based on the fact that after constraining indexes, they share an ordering. | `on` | Yes | Yes |
 | <a name="enforce-home-region"></a> `enforce_home_region` | If set to `on`, queries return an error and in some cases a suggested resolution if they cannot run entirely in their home region. This can occur if a query has no home region (for example, if it reads from different home regions in a [regional by row table](multiregion-overview.html#regional-by-row-tables)) or a query's home region differs from the [gateway](architecture/life-of-a-distributed-transaction.html#gateway) region. Note that only tables with `ZONE` [survivability](when-to-use-zone-vs-region-survival-goals.html) can be scanned without error when this is enabled. For more information about home regions, see [Table localities](multiregion-overview.html#table-localities).<br/><br/>This feature is in preview. It is subject to change. | `off` | Yes | Yes |
+| <a name="enforce-home-region-follower-reads-enabled"></a> `enforce_home_region_follower_reads_enabled` | If `on`, allows the [`enforce_home_region`](cost-based-optimizer.html#control-whether-queries-are-limited-to-a-single-region) setting to perform `AS OF SYSTEM TIME` [follower reads](follower-reads.html) to determine a query's [home region](multiregion-overview.html#table-localities), if any. | `off` | Yes | Yes |
```

Comment:
Maybe change this to be more similar to https://github.com/cockroachdb/cockroach/blob/3042c4fee9457b4f05318fcfa9836433203430cb/pkg/sql/sessiondatapb/local_only_session_data.proto#L346-L348

Either use "determine and report" or "detect and report", or something similar in place of "determine", and also mention that `enforce_home_region` must also be set to `on` for this setting to have any effect.

Reply:
Also, would this be the place to mention that `enforce_home_region_follower_reads_enabled` is a preview feature?

Reply:
Done and done - thanks for pointing out that this is in preview! PTAL.